src/main/java/io/curity/identityserver/plugin/config/PollingPluginConfig.java

```diff
@@ -33,6 +33,10 @@ public interface PollingPluginConfig extends Configuration
     @Description("Maximum time (in seconds), the authenticator waits for a response")
     int getPollMaxWaitTime();
 
+    @DefaultInteger(2)
+    @Description("Poll interval (in seconds)")
+    int getPollingInterval();
```

Comment:
The poller in curity.js has automatic back of, where it increases the poll interval with 1 second every third poll. So this is more of a initial poll interval